lib/purchases_flutter.dart

```diff
@@ -248,13 +248,15 @@ class Purchases {
     return await _channel.invokeMethod('getAppUserID') as String;
   }
 
+  /// Deprecated in favor of logIn.
```

Comment:
this deprecation was missing here too. At one point, we were planning on removing this method entirely, but we deprecated it because we didn't want to do a major.